asciidoc-confluence-publisher-maven-plugin/src/main/java/org/sahli/asciidoc/confluence/publisher/maven/plugin/AsciidocConfluencePublisherMojo.java

```diff
@@ -142,6 +161,42 @@ public void execute() throws MojoExecutionException {
         }
     }
 
+  private void applyUsernameAndPasswordFromSettings() throws MojoExecutionException {
```

Comment:
Do you see any possibility to test this code, preferably in an integration test?